unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsData/QualitySettings.cs

```diff
@@ -78,5 +85,8 @@ public enum ReflectionResolution
         public int maxHQAvatars;
 
         public ReflectionResolution reflectionResolution;
+
+        [Tooltip("Outline Shader Quality")] public OutlineShaderQuality outlineShaderQuality;
```

Comment:
Can this be simplified to "shader quality"? So we can re-use the same config for future shaders

Reply:
Yes, I was thinking about this, there I renamed everything.